examples/basic/layouts/sizing_mode_multiple.py

```diff
@@ -22,17 +22,14 @@
 
 callback = CustomJS(args=dict(source=source, amp=amp, freq=freq, phase=phase, offset=offset),
                     code="""
-    const data = source.data;
     const A = amp.value;
     const k = freq.value;
     const phi = phase.value;
     const B = offset.value;
-    const x = data['x']
-    const y = data['y']
-    for (let i = 0; i < x.length; i++) {
-        y[i] = B + A*Math.sin(k*x[i]+phi);
-    }
-    source.change.emit();
+
+    const x = source.data.x
+    const y = Array.from(x, (x) => B + A*Math.sin(k*x+phi))
+    source.data = { x: x, y: y }
```

Comment:
@mattpap at least for me locally this example was broken with a "squished" like we've seen elsewhere recently